ethcore/call-contract/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "call-contract"
```

Comment:
I wonder if it should be called `ethcore-call-contract` for consistency. While I don't like `ethcore-` prefix, I think it's good to prepend our internal crates with something. Any second opinion on this?

Reply:
Second this!

Reply:
```suggestion
name = "ethcore-call-contract"
```